freud/common.pyx

```diff
@@ -125,6 +133,70 @@ cdef class Compute:
         return wrapper
 
 
+cdef class PairCompute(Compute):
+    R"""Parent class for all compute classes in freud that depend on finding
+    nearest neighbors.
```

Comment:
```suggestion
    neighbor pairs.
```

Reply:
I don't know if neighbor pairs is quite accurate, it's really about finding all the nearest neighbors (by whatever specification) of each particle. It's not just a 1-1 mapping.